src/services/article.ts

```diff
@@ -32,6 +34,32 @@ export default (articleRepo: ArticleRepository, changesRepo: ChangeRepository):
       const changes = await changesRepo.getAllRawChanges(articleId) as JSONObject[];
       manuscript = applyChangesToManuscript(manuscript, changes);
       return serializeManuscript(article, manuscript);
+    },
+
+    getManifest: async (articleId: string) => {
+      const article = await articleRepo.getByArticleId(articleId);
+      if (!article) {
+        return null;
+      }
+
+      let manuscript = getArticleManuscript(article);
+      const changes = await changesRepo.getAllRawChanges(articleId) as JSONObject[];
+      manuscript = applyChangesToManuscript(manuscript, changes);
+
+      const assets = Object.entries(getAllFigureAssets(manuscript)).map(([id, fileName]) => {
+        return {
+          id,
+          type: "image/jpeg",
+          path: `/api/v1/articles/${articleId}/assets/${fileName}`,
```

Comment:
I wonder, if we need to put the fully qualified url here, as otherwise how would any partner who is sent the manifest going to know which hostname to visit. Alternatively, we might want to prefix the hostname later just before the request to Kriya is created.

Reply:
the host name is be the same as the one the partner would have used to call the endpoint

Reply:
We might be limiting the configurability of the app for others by making that decision now though. Sending an absolute url allows others & us configure this in future to a standalone asset hosting service.

Reply:
Yes good point